Modules/@babylonjs/react-native-windows/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -168,4 +170,22 @@ namespace winrt::BabylonReactNative::implementation {
     {
         BabylonNative::RenderView();
     }
+
+    void EngineView::UpdateProperties(IJSValueReader const& reader)
+    {
+        auto const& propertyMap = JSValueObject::ReadFrom(reader);
+
+        for (auto const& pair : propertyMap)
+        {
+            auto const& propertyName = pair.first;
+            auto const& propertyValue = pair.second;
+
+            if (propertyName == "isTransparent")
+            {
+                bool isTransparent = propertyValue.AsBoolean();
+                auto engineView = static_cast<winrt::Windows::UI::Xaml::Controls::SwapChainPanel>(*this);

```

Comment:
Why is this cast needed?

Reply:
This is also unsafe with C++/WinRT. If you need to cast between types always use the `as` or `try_as` functions.

But since `EngineView` derives from `SwapChainPanel` you should be able to just directly call `CompositeMode`. IntelliSense might complain because of C++/WinRT's extreme usage of CRTP, but I think it should still compile.

Reply:
cast removed.